gtsfm/evaluation/visualize_benchmark_comparison.py

```diff
@@ -0,0 +1,254 @@
+"""
+Use Plotly to create a heatmap representing a 2d table, and add annotations to it.
```

Comment:
Maybe add that it compares the metrics across all the benchmarks from the CI? Currently lacks any description of what the 2d table is.